.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - run: pip install --user coverage
       - run: pip install --user -e .
       - run: pip install --user flake8
-      - run: python -m flake8
+      - run: python -m flake8 --extend-ignore=F401
```

Comment:
Why do we need to add this exception?

Reply:
flake8 complains when we import something and do not use it. This is the case in the file we're adding, marqeuz_client/__init__.py.